cmd/validate/image.go

```diff
@@ -318,7 +320,14 @@ func validateImageCmd(validate imageValidationFunc) *cobra.Command {
 					log.Debugf("policySource: %#v", policySource)
 				}
 
-				c, err := newConftestEvaluator(cmd.Context(), policySources, data.policy, sourceGroup)
+				var c evaluator.Evaluator
+				var err error
+				if os.Getenv("EC_USE_OPA") != "" {
```

Comment:
Maybe this:
```suggestion
				if os.Getenv("EC_USE_OPA") != "1" {
```

Since it I might do this `export EC_USE_OPA=0` or `EC_USE_OPA=no` and expect it to dwim.

Reply:
I used "true" 
```golang
if os.Getenv("EC_USE_OPA") != "true" {
```

Reply:
Good idea. Hadn't noticed that.